src/bulletin/bulletin.service.ts

```diff
@@ -69,4 +69,98 @@ export class BulletinService {
       }
     });
   }
+
+  async updateBulletin(
+    id: number,
+    updateBulletinDto: BulletinDTO,
+    pdfAttachments: Express.Multer.File[],
+  ) {
+    return this.prismaService.$transaction(async (tx) => {
+      try {
+        const updatedBulletin = await tx.bulletin.update({
+          where: { id },
+          data: {
+            category_id: updateBulletinDto.category_id,
+            title: updateBulletinDto.title,
+            content: updateBulletinDto.content,
+            author: updateBulletinDto.author,
+          },
+        });
+
+        if (pdfAttachments && pdfAttachments.length > 0) {
```

Comment:
create isEmpty util function

Reply:
I feel like this part of the code unecessary to create a util function, because still readable and only a single line of code. Considering this one.